src/main/scala/org/grapheco/lynx/dataframe/DefaultDataFrameOperator.scala

```diff
@@ -89,25 +89,24 @@ class DefaultDataFrameOperator(expressionEvaluator: ExpressionEvaluator) extends
 
   override def orderBy(df: DataFrame, sortItem: Seq[(Expression, Boolean)])(ctx: ExpressionContext): DataFrame = {
     val columnsName = df.columnsName
-    DataFrame(df.schema, () => df.records.toSeq
-      .sortWith { (A, B) =>
-        val ctxA = ctx.withVars(columnsName.zip(A).toMap)
-        val ctxB = ctx.withVars(columnsName.zip(B).toMap)
-        val sortValue = sortItem.map {
-          case (exp, asc) =>
-            (expressionEvaluator.eval(exp)(ctxA), expressionEvaluator.eval(exp)(ctxB), asc)
-        }
-        _ascCmp(sortValue.toIterator)
-      }.toIterator)
+    val sortedRecords = df.records.toSeq.sortWith { (A, B) =>
+      val ctxA = ctx.withVars(columnsName.zip(A).toMap)
+      val ctxB = ctx.withVars(columnsName.zip(B).toMap)
+      val sortValues = sortItem.map { case (exp, asc) =>
+        val valueA = expressionEvaluator.eval(exp)(ctxA)
+        val valueB = expressionEvaluator.eval(exp)(ctxB)
+        (valueA.compareTo(valueB), asc)
+      }
+      sortValues.exists { case (cmp, asc) => cmp != 0 && (cmp > 0) == asc }
```

Comment:
It doesn't seem to make much sense here. Need to return asc judgement on the first different value. For example the following example: `[cmp,asc]={(0,t),(1,f),(1,t)}`. should return `false` at `(1,f)`, but the exists function will return true, because `(1,t)`.